### DISCUSSION TITLE
[9.5.3 - basePath & Rewrites] Doesn't work when basePath set to false

### DISCUSSION THREAD

**Discussion by mrgoonie** [vercel/next.js#16958] [Help]
I'm following this example: https://nextjs.org/docs/api-reference/next.config.js/rewrites#rewrites-with-basepath-support

// next.config.js
```
module.exports = {
  basePath: "/app",
  async rewrites() {
    return [
          {
            source: "/:slug*",
            destination: "/app/:slug*",
            basePath: false,
          }
        ];
  },
};
```

Then run `yarn dev` and this error was shown in the terminal:
> The route /:slug* rewrites urls outside of the basePath. Please use a destination that starts with `http://` or `https://` https://err.sh/vercel/next.js/invalid-external-rewrite.md

Not sure what I have done wrong...

**Comment by IikkaWinter** [comment]
This still persists in 10.0.

**Comment by nicolascbarbosa** [comment]
Same thing here :cry:

**Comment by dcorb** [comment]
The documentation seems to be wrong. I found this commit, where they prevent the usage of basePath false with internal destination
https://github.com/vercel/next.js/commit/92304404a4cb5589efc199b611158540565a6959

**Comment by ijjk** [comment]
Hi, good catch the documentation does appear to be incorrect here, I opened a PR here https://github.com/vercel/next.js/pull/20663 to update this. Rewriting to internal resources with `basePath: false` is not allowed since it can lead to duplicate content and defeats the purpose of the `basePath` to allow rewriting outside of the `basePath`.
